modis_tools/resources.py

```diff
@@ -21,17 +21,14 @@ def __init__(
     ):
         super().__init__(session=session, username=username, password=password)
 
-    @params_args(CollectionDoiParams)
-    def query(
-        self, short_name: Optional[str] = None, version: Optional[str] = None, **kwargs
-    ) -> List[Collection]:
+    def query(self, **kwargs) -> List[Collection]:
         params = kwargs.pop("params", {})
         params = {**(self.params or {}), **kwargs, **params}
```

Comment:
Are these two lines of still necessary after removing the decorator? My understanding is that the `params` item in the `kwargs` is created by the decorator